modules/history/src/views/full/style.scss.d.ts

```diff
@@ -3,7 +3,8 @@
 interface CssExports {
   'conversations': string;
   'datepicker': string;
-  'daypicker-item': string;
+  'displayFlagCheckbox': string;
+  'downloadCopy': string;
```

Comment:
let's use camelCase or hypen-case (I don't care at all) but not both :-)

Reply:
Yeah I know. I started the history module with hypen-case because that's I was used to, but then Yan told me to use camelCase for the new stuff. So the hypen-case classes are only older classes I have not refactored yet.